<launch>
  <group ns="vision">
    <node pkg="nodelet" type="nodelet" name="vision_nodelet"  args="manager"/>
    <node pkg="nodelet" type="nodelet" name="camera" args="load uvc_camera/StereoNodelet vision_nodelet">
    	<param name="right/device" type="string" value="/dev/video1" />
      <param name="left/device" type="string" value="/dev/video0" />
      <param name="width" type="int" value="1280" />
      <param name="height" type="int" value="720" />
    </node>
    <remap from="yolo2/left/image" to="left/image_raw" />
    <remap from="yolo2/right/image" to="right/image_raw" />
    <node pkg="nodelet" type="nodelet" name="yolo2" args="load yolo2/Yolo2Nodelet vision_nodelet" output="screen">
      <param name="confidence" value="0.5" />
      <param name="nms" value="0.4" />
    </node>
  </group>
</launch>
